docker/linux/base/Dockerfile

```diff
@@ -15,6 +15,20 @@ RUN apt update -qqy \
         python3-setuptools \
         git ccache \
         curl xz-utils \
+        build-essential \
+        libssl-dev \
+        libcurl4-gnutls-dev \
+        libexpat1-dev \
+        gettext \
+        unzip \
+        zlib1g-dev \
+    && curl -fsSL "https://github.com/git/git/archive/v2.18.0.tar.gz" -o git-2.18.0.tar.gz \
```

Comment:
I would add checksum check like we do here: https://lbshell-internal.git.corp.google.com/cobalt_src/+/refs/heads/COBALT/docker/linux/base/build/Dockerfile#55